lib/overcommit/hook/pre_commit/yaml_lint.rb

```diff
@@ -11,7 +11,11 @@ def run
       if result.success?
         :pass
       else
-        return [:warn, result.stdout]
+        if result.stdout.include? "error"
+          return [:fail, result.stdout]
+        else
+          return [:warn, result.stdout]
+        end
       end
```

Comment:
The test failure is because the way this `if`/`else` is written could be simplified.

You can resolve by writing as:
```ruby
if result.success?
  :pass 
elsif result.stdout.include?("error")
  [:fail, result.stdout]
else
  [:warn, result.stdout]
end
```

Notice this is "flatter" than what is currently written.